testing/integration/logs_ingestion_test.go

```diff
@@ -125,7 +125,75 @@ func TestDebLogIngestFleetManaged(t *testing.T) {
 	// name. This policy does not contain any integration.
 	t.Log("Enrolling agent in Fleet with a test policy")
 	createPolicyReq := kibana.AgentPolicy{
-		Name:        fmt.Sprintf("test-policy-enroll-%d", time.Now().Unix()),
+		Name:        fmt.Sprintf("test-policy-enroll-%s", uuid.New().String()),
```

Comment:
Thank you!
Related to https://github.com/elastic/elastic-agent/pull/4421
I guess this test was not there when I fixed it 🙂